### PR TITLE
Dev stage merge

### PR DESCRIPTION
**Pull request by ravinderwalkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/all-modules/all-modules.component.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    public loadAddManageComponent(): void {
+        let componentFactory = this.componentFactoryResolver.resolveComponentFactory(ManageGroupsAccountsComponent);
+        let viewContainerRef = this.addmanage.viewContainerRef;
+        viewContainerRef.clear();
+        let componentRef = viewContainerRef.createComponent(componentFactory);
+        (componentRef.instance as ManageGroupsAccountsComponent).closeEvent.subscribe((a) => {
+            this.hideManageGroupsModal();
+            viewContainerRef.remove();
+        });
+        this.manageGroupsAccountsModal.onShown.subscribe((a => {
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/all-modules/all-modules.component.ts:68]: rename 'a'

File: apps/web-giddh/src/app/all-modules/all-modules.component.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+     * This will show the manage groups and account popup
+     *
+     * @memberof AllModulesComponent
+     */
+    public loadAddManageComponent(): void {
+        let componentFactory = this.componentFactoryResolver.resolveComponentFactory(ManageGroupsAccountsComponent);
+        let viewContainerRef = this.addmanage.viewContainerRef;
+        viewContainerRef.clear();
+        let componentRef = viewContainerRef.createComponent(componentFactory);
+        (componentRef.instance as ManageGroupsAccountsComponent).closeEvent.subscribe((a) => {
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/all-modules/all-modules.component.ts:64]: rename 'a'

File: apps/web-giddh/src/app/contact/contact.component.scss [hunk 3]
```diff
@@ -341,14 +326,14 @@ span.inputAddon {
     .onDesktopView.btn-group {
         display: none;
     }
-    .custumerRightWrap {
+    .customer-right-wrap {
         padding-right: 0;
         position: fixed;
         bottom: 0;
         width: 100%;
         background: #fff;
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/contact/contact.component.scss:334]: use variable for         background: #fff;

File: apps/web-giddh/src/app/daybook/daybook.component.scss [hunk 4]
```diff
@@ -46,9 +47,9 @@
     align-items: center;
     font-size: 18px;
     margin-left: 7px;
-    border: 1px solid #d9d9d9;
+    border: 1px solid $gainsboro;
     border-radius: 2px;
-    color: #1a237e;
+    color: $midnight-blue;
     margin-right: 7px;
     background: #fff;
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/daybook/daybook.component.scss:54]: Use variable for   background: #fff;

File: apps/web-giddh/src/app/invoice/proforma/proforma-list.component.html [hunk 5]
```diff
@@ -108,13 +89,76 @@
                 </div>
             </div>
         </div>
+        <div class="d-flex mt-1">
+            <div *ngIf="voucherType === 'proformas'" class="btn-wrapper mr-15">
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/invoice/proforma/proforma-list.component.html:93]: Use VoucherTypeEnum ''proformas''

File: apps/web-giddh/src/app/invoice/proforma/proforma-list.component.html [hunk 6]
```diff
@@ -108,13 +89,76 @@
...
+                <button class="btn btn-pink" style="margin: 0 !important;" (click)="goToInvoice('proformas')">+ New
+                    Proforma
+                </button>
+            </div>
+
+            <div *ngIf="voucherType === 'estimates'" class="btn-wrapper mr-15">
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/invoice/proforma/proforma-list.component.html:99]: Use VoucherTypeEnum ''estimates''

File: apps/web-giddh/src/app/contact/contact.component.scss (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/contact/contact.component.scss:334]: not worked on particular line, please focus on red and green color line

File: apps/web-giddh/src/app/daybook/daybook.component.scss (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/daybook/daybook.component.scss:54]: not worked on particular line, please focus on red and green color line

File: apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html [hunk 7]
```diff
@@ -1142,6 +1142,10 @@
                                         </div>
                                     </div>
 
+<<<<<<< HEAD
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html:1145]: Please check resolve this conflict

File: apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html [hunk 8]
```diff
@@ -1142,6 +1142,10 @@
...
+                    <!--region balance due for non-purchase-->
+                    <div class="tableRow" *ngIf="((isSalesInvoice || isCashInvoice)) && !(!(isAccountHaveAdvanceReceipts || isInvoiceAdjustedWithAdvanceReceipts) && isUpdateMode)">
+=======
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html:1148]: Please check resolve this conflict

File: apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html [hunk 9]
```diff
@@ -1152,13 +1156,23 @@
                                         <div class="tableCell figureCell"><span *ngIf="isMulticurrencyAccount && !isPurchaseInvoice">{{invFormData.accountDetails.currencySymbol}}</span>{{invFormData.voucherDetails.tdsTotal | giddhCurrency}}
                                         </div>
                                     </div>
+>>>>>>> 99d096c86784e074bef714318ec0bc434b4b57f5
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html:1160]: Please check resolve this conflict

File: apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html [hunk 10]
```diff
@@ -1152,13 +1156,23 @@
...
 
                                     <!--region balance due for non-purchase-->
                                     <div class="tableRow" *ngIf="((isSalesInvoice || isCashInvoice)) && !(!(isAccountHaveAdvanceReceipts || isInvoiceAdjustedWithAdvanceReceipts) && isUpdateMode)">
 
                                         <div class="tableCell depositeSection">
 
+<<<<<<< HEAD
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html:1166]: Please check resolve this conflict

File: apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html [hunk 11]
```diff
@@ -1205,10 +1219,26 @@
                                                                                         {{adjustPaymentData.totalAdjustedAmount | giddhCurrency}}
                                                                                     </p>
                                                                                 </span>
+<<<<<<< HEAD
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.html:1222]: Please check resolve this conflict

**Pull request closed without merging**